content/sensu-go/5.16/api/namespaces.md

```diff
@@ -130,7 +165,8 @@ The `/user-namespaces` API endpoint provides HTTP GET access to the namespaces t
 The following example demonstrates a request to the `/user-namespaces` API endpoint, resulting in a JSON array that contains the namespaces the user has access to.
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/enterprise/user-namespaces -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/enterprise/user-namespaces
```

Comment:
requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency